pyproject.toml

```diff
@@ -47,6 +47,8 @@ version = "0.9.35"
 [tool.poetry.dependencies]
 SQLAlchemy = {extras = ["asyncio"], version = ">=1.4.49"}
 beautifulsoup4 = {optional = true, version = "^4.12.2"}
+botocore = ">=1.33.13"
```

Comment:
might not want to add dep here for those 2

Reply:
As they are not lazily imported, they should be added; otherwise, tests will generate errors.

Reply:
oh you mean unit tests?

Reply:
Both can be in dev dependencies, no need to add to core deps